client/signup/steps/domains/index.jsx

```diff
@@ -1,61 +1,60 @@
 /**
  * External dependencies
  */
-import React from 'react';
-import PropTypes from 'prop-types';
-import { connect } from 'react-redux';
-import { defer, get, includes, isEmpty } from 'lodash';
-import { localize, getLocaleSlug } from 'i18n-calypso';
-import classNames from 'classnames';
-
+import CartData from 'calypso/components/data/cart';
```

Comment:
Not sure why this was changed, it will be nice to maintain previous order so it is easier to review.

Reply:
Hmmm there was a merge conflict and I just vscode formatted imports :(
I'll see if I can fix this.
CC - @niranjan-uma-shankar

Reply:
Fixed